utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -83,9 +83,19 @@
     public final static int DEFAULT_AUTOSCALE_POLICY_INTERVAL_TIME = 30;
     public final static int DEFAULT_AUTOSCALE_POLICY_QUIET_TIME = 5 * 60;
     private final static Random s_rand = new Random(System.currentTimeMillis());
-
-    public static long createSequenceBasedMacAddress(final long macAddress) {
-        return macAddress | 0x060000000000l | (long)s_rand.nextInt(32768) << 25 & 0x00fffe000000l;
+    private final static long prefix = 0x1e;
+
+    public static long createSequenceBasedMacAddress(final long macAddress, long globalConfig) {
```

Comment:
It is not possible for macAddress, globalConfig to less than zero. 
For globalconfig maximum value 255 is allowed. 
macAddress value is coming from the DB (ipVO)